nav2_costmap_2d/include/nav2_costmap_2d/costmap_2d_ros.hpp

```diff
@@ -89,6 +89,18 @@ class Costmap2DROS : public nav2_util::LifecycleNode
    */
   explicit Costmap2DROS(const std::string & name, const bool & use_sim_time = false);
 
+  /**
+   * @brief  Constructor for the wrapper, the node will
+   * be placed in a namespace equal to the parent_namespace and the node's name
+   * @param name Name of the costmap ROS node
+   * @param name Absolute namespace of the node hosting the costmap node
+   * @param use_sim_time Whether to use simulation or real time
+   */
+  explicit Costmap2DROS(
```

Comment:
Can we remove this? We have alot of constructors already and if this is just for a test, we can use the one with the parent / local namespaces and just leave the local namespace = name

Reply:
Reverted in https://github.com/ros-navigation/navigation2/pull/4715/commits/ac5d861dbf44f0c80bd624c4d55722e120c12c28